### PR TITLE
DOCSP-41120: connection landing

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin-sync/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-41120>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/kotlin-sync/DOCSP-41120-cxn/connect/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/connect.txt [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+----------
+
+The following sections describe how to connect to different targets,
+such as a local instance of MongoDB or a cloud-hosted instance on Atlas.
+
+Local Deployment
+~~~~~~~~~~~~~~~~
+
+The following code shows the connection string to connect to a local
+instance of MongoDB:
```
Comment (mcmorisi) [inline source/connect.txt:71]: Can we do intros for all the examples? Either that or removing this one to keep consistency with the other sections.

File: source/connect.txt [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/connect/disable-host-verification-tabs.rst
+
+.. To learn more about disabling hostname verification, see :ref:`kotlin-sync-insecure-tls` in
+.. the TLS configuration guide.
+
+Network Compression
+-------------------
+
```
Comment (mcmorisi) [inline source/connect.txt:128]: I think we should include a brief intro for the whole section (similar to how the TLS section has a quick sentence to start).

File: source/connect.txt [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                   "zlibCompressionLevel=<zlib compression level>"
+         
+         val mongoClient = MongoClient.create(uri)
+
+.. To learn more about setting the zlib compression level, see
+.. :ref:`kotlin-sync-enable-compression` in the Network Compression guide.
+
+Server Selection
+----------------
+
```
Comment (mcmorisi) [inline source/connect.txt:172]: Same suggestion as the Network Compression section

File: source/connect.txt [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+   val client = MongoClient.create("mongodb://<username>:<password>@<hostname>:<port>",
+                                server_selector=<selector function>)
+
+.. To learn more about customizing server selection, see
+.. :ref:`kotlin-sync-server-selection`.
+
+{+stable-api+}
+--------------
+
```
Comment (mcmorisi) [inline source/connect.txt:183]: Same suggestion as the Network Compression section

**Review by mcmorisi — CHANGES_REQUESTED** [review]
Some suggestions, mostly about adding a little more introduction to sections and examples. Good work!

File: source/connect.txt (hunk 1 above)
Comment (rustagir) [reply source/connect.txt:71]: Yep! Got caught up in the copy paste

**Review by mcmorisi — APPROVED** [review]
LGTM with two nits, thanks for the quick follow!

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**